### PR TITLE
Supporting Customer-managed keys for managed disks in VMWare to Azure and HyperV to Azure scenarios

### PR DESCRIPTION
**Pull request by TalluriAnusha**
Supporting Customer-managed keys for managed disks in VMWare to Azure and HyperV to Azure scenarios

### CODE REVIEW

**Comment by TalluriAnusha** [comment]
Swagger approval is done. 
PR link : https://github.com/Azure/azure-rest-api-specs/pull/7961

**Comment by TalluriAnusha** [comment]
https://github.com/Azure/azure-rest-api-specs/pull/7698
https://github.com/Azure/azure-rest-api-specs/pull/7982

File: sdk/recoveryservices-siterecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Recovery Services SiteRecovery Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Recovery Services SiteRecovery Resources.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.3.0")]
+[assembly: AssemblyVersion("2.0.4.0")]
```
Comment (isra-fel) [inline sdk/recoveryservices-siterecovery/Microsoft.Azure.Management.RecoveryServices.SiteRecovery/src/Properties/AssemblyInfo.cs]:
```suggestion
[assembly: AssemblyVersion("2.0.0.0")]
```
Keep assembly version unchanged until a breaking change

**Review by isra-fel — CHANGES_REQUESTED** [review]
One line needs to be changed

**Review by isra-fel — APPROVED** [review]

**Pull request merged**